packages/ERTP/src/types-ambient.js

```diff
@@ -22,8 +17,8 @@
  */
 
 /**
- * @typedef {NatValue | SetValue | CopySet | CopyBag} AmountValue An
- *   `AmountValue` describes a set or quantity of assets that can be owned or
+ * @typedef {NatValue | SetValue | CopySet | import('@endo/patterns').CopyBag} AmountValue
```

Comment:
Why does `CopyBag` need to be imported but `CopySet` does not?

Reply:
Because @agoric/store has an old typedef for CopySet but not for CopyBag. Eventually, we should import both from @endo/patterns. Eventually, @agoric/store should stop reexporting things that should be imported from elsewhere.